src/components/CompareResults/ResultsTable.tsx

```diff
@@ -104,9 +104,17 @@ export default function ResultsTable() {
   const [searchTerm, setSearchTerm] = useState(initialSearchTerm);
   const [frameworkIdVal, setFrameworkIdVal] = useState(frameworkId);
   const [tableFilters, setTableFilters] = useState(
-    new Map() as Map<string, Set<string>>, // ColumnID -> Set<Values to remove>
+    new Map([['confidence', new Set(['High'])]]), // ColumnID -> Set<Values to remove>// Default confidence filter
   );
 
+  useEffect(() => {
+    setTableFilters((prevFilters) => {
+      const newFilters = new Map(prevFilters);
+      newFilters.set('confidence', new Set(['High']));
+      return newFilters;
+    });
+  }, []);
```

Comment:
can you please explain why you used this effect in addition to the default state above?
Thanks